src/main/java/core/basesyntax/exception/NoAnnotationsExceptions.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.exception;
+
+public class NoAnnotationsExceptions extends Exception {
```

Comment:
```suggestion
public class NoAnnotationsExceptions extends RuntimeException {
```
і давай видалимо try-catch в мейні

Reply:
done